src/multi-level-checkbox-editor/multi-level-checkbox-editor.tsx

```diff
@@ -1,8 +1,12 @@
 import * as React from 'react';
 
+import { matches } from '@ampel-ui/common/search';
```

Comment:
Shouldn't it be configurable?!

Reply:
Well It can be, but as far I know that's not common behaviour in other components as well (table, selects with search etc.)

Reply:
Discussed in person.